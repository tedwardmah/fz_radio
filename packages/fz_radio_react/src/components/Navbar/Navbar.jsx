import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import MenuIcon from '@material-ui/icons/Menu';

function Navbar() {
  return (
    <AppBar position="static">
      <Toolbar>
        <IconButton color="inherit" aria-label="Menu">
          <MenuIcon />
        </IconButton>
        <Typography variant="h6" color="inherit">
          Friend Zone Radio
        </Typography>
      </Toolbar>
    </AppBar>
  );
}

export default Navbar;
