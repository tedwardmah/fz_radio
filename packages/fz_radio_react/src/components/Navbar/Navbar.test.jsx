import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './Navbar';

it('renders without crashing', () => {
  const wrapper = shallow(<Navbar />);
  expect(wrapper).toMatchSnapshot();
});
