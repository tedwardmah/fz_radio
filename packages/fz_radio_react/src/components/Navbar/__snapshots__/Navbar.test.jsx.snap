// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<WithStyles(AppBar)
  position="static"
>
  <WithStyles(Toolbar)>
    <WithStyles(IconButton)
      aria-label="Menu"
      color="inherit"
    >
      <pure(MenuIcon) />
    </WithStyles(IconButton)>
    <WithStyles(Typography)
      color="inherit"
      variant="h6"
    >
      Friend Zone Radio
    </WithStyles(Typography)>
  </WithStyles(Toolbar)>
</WithStyles(AppBar)>
`;
